diff --git a/android/app/build.gradle b/android/app/build.gradle
index 3510e0a..75d3331 100644
--- a/android/app/build.gradle
+++ b/android/app/build.gradle
@@ -93,6 +93,9 @@ android {
         targetSdkVersion rootProject.ext.targetSdkVersion
         versionCode 1
         versionName "1.0.0"
+        manifestPlaceholders = [
+            appAuthRedirectScheme: 'io.identityserver.demo'
+        ]
     }
     signingConfigs {
         debug {
